gitpod-io/openvscode-server
# Improve ports

## Related issues
Fix https://github.com/gitpod-io/gitpod/issues/8377
Fix https://github.com/gitpod-io/gitpod/issues/11616
## How to test

1. Open this PR in vscode-desktop, compile (run `yarn vscode:prepublish` in gitpod-remote folder) and launch "Launch gitpod-remote" debug config
2. Open a workspace with ports configured in gitpod.yml
3. Check the ports are forwarded when connecting in ports view
4. Run some command that uses some port
5. Check the port is forwarded automatically
6. Check exposed ports webview view is visible and working as expected
